op-node/rollup/derive/l1_block_info.go

```diff
@@ -105,7 +115,7 @@ func (info *L1BlockInfo) marshalBinaryBedrock() ([]byte, error) {
 
 func (info *L1BlockInfo) unmarshalBinaryBedrock(data []byte) error {
 	if len(data) != L1InfoBedrockLen {
-		return fmt.Errorf("data is unexpected length: %d", len(data))
+		return fmt.Errorf("data is unexpected length: %d %d", len(data), L1InfoBedrockLen)
```

Comment:
The error message could be better here: `data is unexpected length: got %d, expected %d`